delta/plugins/archive/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/archive/model/ArchiveRejection.scala

```diff
@@ -166,6 +166,7 @@ object ArchiveRejection {
     case JsonLdRejection.UnexpectedId(id, sourceId)        => UnexpectedArchiveId(id, sourceId)
     case JsonLdRejection.InvalidJsonLdFormat(id, rdfError) => InvalidJsonLdFormat(id, rdfError)
     case JsonLdRejection.DecodingFailed(error)             => DecodingFailed(error)
+    case JsonLdRejection.BlankId                           => InvalidArchiveId("")
```

Comment:
Question: all these InvalidId rejections here and below, they are defined to handle the case where the provided `@id` cannot be resolved/expanded to an IRI. Isn't it possible that the empty string can be resolved, and it is just that we decide not to allow it? Then the error would not be entirely correct.

Reply:
This is now a new error